jetbrains-rider/tst/base/AwsReuseSolutionTestBase.kt

```diff
@@ -45,6 +49,15 @@ abstract class AwsReuseSolutionTestBase : BaseTestWithSolutionBase() {
         allowCustomDotnetRoots()
     }
 
+    @BeforeClass(alwaysRun = true)
```

Comment:
Why do we provide this flag explicitly(in the previous After tag as well)? Is the default false?(that's surprising)

Reply:
testng default behavior is to skip the after* methods if the test fails. for the before* methods this forces the method to be run regardless of what test group it's in